### PR TITLE
Support concourse pipeline for stampy

### PR DESCRIPTION
**Pull request by andreas-kupries**
1. Manually vendored test dependencies
1. Reworked non-standard `make/build` and `make/package` to match `fissile`s layout (`build` + arch subdirs).

### CODE REVIEW

**Review by mook-as — APPROVED** [review]
Mostly looks fine, but we should have a record of _which_ packages we vendored (mostly in commit hash references).  Tools like govendor/glide/whatever tend to do that, but it looks like at this point everything's limping along and waiting for `dep`.

File: make/build [hunk 1]
```diff
@@ -11,6 +11,14 @@ set +o errexit
 set -o errexit
 set -o nounset
 
+OSES="linux darwin"
+GOARCH=${GOARCH:-$(go env GOARCH)}
+
 for exe in stampy graphy; do
-  go build -ldflags="-X main.version=${APP_VERSION}" "./cmd/${exe}/"
+  for OS in ${OSES}; do
+      GOOS="${OS}" go build \
```
Comment (mook-as) [inline make/build]: Do we actually need stampy/graphy for mac?

Comment (andreas-kupries) [inline make/build]: With `make vagrant-prep` now able to run on the dev host instead of just inside a vagrant I believe yes.

**Comment by andreas-kupries** [comment]
Re vendoring - I agree that it should have done with a tool. Recalling the trouble these tools are / can be I went for the quick solution to get me to testing.

I am willing to have merging defered until the vendoring is done right, with a tool.
If you really want to merge without that, then please make a ticket recording our need to fix it.

**Comment by mook-as** [comment]
Ah, if you're up for it, then please choose a tool (or make @aarondl choose one) and use it before we merge 🙂

**Comment by mook-as** [comment]
FWIW, with the vendoring in place, `make test` actually tests the vendored stuff too. Might need to copy how fissile does it or something to fix that.

**Comment by andreas-kupries** [comment]
Uh :( Will fix.

**Comment by andreas-kupries** [comment]
Fixed testing with commit 81aacced5c3488d0b149b5dc315199bf066ac549

**Review by mook-as — APPROVED** [review]

**Comment by mook-as** [comment]
Gave up on waiting for production-concourse, and deployed to a test concourse instance instead. Looks good.

**Pull request merged**